### DISCUSSION TITLE
being stuck at flowise installation

### DISCUSSION THREAD

**Discussion by simonator1001** [FlowiseAI/Flowise#3167] [Q&A]
Trying to install flowise via coolify on my server but it kept stuck at  the following step for like over 12 hours..... not sure if anyone has similar experience and would have any idea on how i might get to resolve this? thanks!

2024-Sep-07 17:34:35.414085
#13 4.375 flowise-ui:build: vite v5.1.6 building for production...
2024-Sep-07 17:34:35.698756
#13 4.660 flowise-ui:build: transforming...

**Comment by taowang1993** [comment]


**Reply by simonator1001** [reply]
awesome, i'll try that out. Thanks a tons!

**Reply by clrt1** [reply]
Did you get a solution to this?

My installation on a Mac is stuck on:
npm warn deprecated puppeteer@20.9.0: < 22.8.2 is no longer supported
npm warn deprecated @oclif/screen@3.0.8: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.

Would appreciate any help, thanks!

**Comment by NoviceArtisan** [comment]
> Trying to install flowise via coolify on my server but it kept stuck at the following step for like over 12 hours..... not sure if anyone has similar experience and would have any idea on how i might get to resolve this? thanks!
> 
> 2024-Sep-07 17:34:35.414085 #13 4.375 flowise-ui:build: vite v5.1.6 building for production... 2024-Sep-07 17:34:35.698756 #13 4.660 flowise-ui:build: transforming...

mines stuck at
npm warn deprecated puppeteer@20.9.0: < 22.8.2 is no longer supported
npm warn deprecated highlight.js@9.15.10: Version no longer supported. Upgrade to @latest

I have no desire to install docker or other such workarounds.

**Comment by Technorocker** [comment]
I am just trying to install this as well and cam looking for fixes. Also not looking to install and setup docker to use this. I am having  the same issue as @NoviceArtisan right now. install has been stuck at 

```
npm warn deprecated gauge@3.0.2: This package is no longer supported.
npm warn deprecated gauge@4.0.4: This package is no longer supported.
npm warn deprecated puppeteer@20.9.0: < 22.8.2 is no longer supported
npm warn deprecated highlight.js@9.15.10: Version no longer supported. Upgrade to @latest
```
for over an hour now.

**Comment by dimemacau** [comment]
Same here stuck at 

npm warn deprecated gauge@4.0.4: This package is no longer supported.
npm warn deprecated are-we-there-yet@3.0.1: This package is no longer supported.
npm warn deprecated npmlog@6.0.2: This package is no longer supported.
npm warn deprecated highlight.js@9.15.10: Version no longer supported. Upgrade to @latest

**Reply by fastscripts** [reply]
I had the same issue. I fixed it by running the installation of highlight.js by hand. npm install highlight.js@9.15.10 and then running npm install flowise again
